sbsigntools-mktarball.sh

```diff
@@ -12,13 +12,12 @@ tmp=$(mktemp -d)
 
 unset CDPATH
 pwd=$(pwd)
-version=0.9.4
-commit=d52f7bbb73401aab8a1d59e8d0d686ad9641035e
+version=$(cat version)
 
 pushd "$tmp"
 git clone git://git.kernel.org/pub/scm/linux/kernel/git/jejb/sbsigntools.git
 cd sbsigntools
-git checkout ${commit}
+git checkout v${version}
```

Comment:
This won't fly, using the commit is intentional because it ensures integrity (you can't push different content while keeping the same commit hash). Unfortunately, that doesn't work for tags. But the tags seems to be signed, so if you add signature verification that could be okay (we don't have fetching from git in any other `get-sources` target, but can take a look at https://github.com/QubesOS/qubes-builder/blob/main/scripts/verify-git-tag#L86-L94 for example.

That said, the `make get-sources` approach works only for legacy qubes-builder, but it isn't supported in qubes-builderv2 (which is used for R4.2 packages). That's the reason for the CI failure. The later uses `.qubesbuilder` and doesn't support fetching from git, only tarballs (either with detached signature or local hash file). The easiest way would be do that, and use the automatic tarball from kernel.org (check locally if it matches the signed tag content and commit hash file to the repository). Some example: https://github.com/QubesOS/qubes-linux-scrypt/blob/main/.qubesbuilder